src/extensions/messages.ts

```diff
@@ -158,4 +181,20 @@ export const messages = defineMessages({
     description: "Installed extensions column name",
     id: "7jxSx8",
   },
+  deleteAppHeader: {
+    defaultMessage: "Delete App",
+    id: "d+5yJO",
+    description: "delete app header",
+  },
+  deleteApp: {
+    id: "6hLZNA",
+    defaultMessage: "Are you sure you want to delete this app?",
+    description: "delete app",
+  },
+  deleteNamedApp: {
+    id: "EWD/wU",
+    defaultMessage:
+      "Deleting {name}, you will remove installation of the app. If you are paying for app subscription, remember to unsubscribe from the app in Saleor Marketplace. Are you sure you want to delete the app?",
+    description: "delete app",
+  },
```

Comment:
issue: Is this copy correct? We're using **App** instead of **Extension**

additional suggestion: in description we should probably use something like `delete extension page`